specification/cognitiveservices/data-plane/Language/preview/2021-11-01-preview/textanalytics.json

```diff
@@ -0,0 +1,2106 @@
+            "type": "object",
+            "description": "Base parameters object for a text analysis task.",
+            "properties": {
+                "loggingOptOut": {
+                    "type": "boolean",
+                    "default": false
+                }
+            }
+        },
+        "PreBuiltTaskParameters": {
+            "type": "object",
+            "description": "Parameters object for a text analysis task using pre-built models.",
+            "properties": {
+                "modelVersion": {
+                    "type": "string",
+                    "default": "latest"
+                }
+            }
+        },
+        "PreBuiltResult": {
+            "properties": {
+                "errors": {
+                    "type": "array",
+                    "description": "Errors by record id.",
+                    "items": {
+                        "$ref": "#/definitions/RecordError"
+                    }
+                },
+                "statistics": {
+                    "$ref": "#/definitions/RequestStatistics"
+                },
+                "modelVersion": {
+                    "type": "string",
+                    "description": "This field indicates which model is used for scoring."
+                }
+            },
+            "required": [
+                "errors",
+                "modelVersion"
+            ]
+        },
+        "CustomTaskParameters": {
+            "type": "object",
+            "description": "Parameters object for a text analysis task using custom models.",
+            "properties": {
+                "project-name": {
+                    "type": "string"
+                },
+                "deployment-name": {
+                    "type": "string"
+                }
+            },
+            "required": [
+                "project-name",
+                "deployment-name"
+            ]
+        },
+        "CustomResult": {
+            "properties": {
+                "errors": {
+                    "type": "array",
+                    "description": "Errors by document id.",
+                    "items": {
+                        "$ref": "#/definitions/RecordError"
+                    }
+                },
+                "statistics": {
+                    "$ref": "#/definitions/RequestStatistics"
+                },
+                "projectName": {
+                    "type": "string",
+                    "description": "This field indicates the project name for the model."
+                },
+                "deploymentName": {
+                    "type": "string",
+                    "description": "This field indicates the deployment name for the model."
+                }
+            },
+            "required": [
+                "errors",
+                "projectName",
+                "deploymentName"
+            ]
+        },
+        "CustomEntitiesTask": {
+            "type": "object",
+            "description": "TODO",
```

Comment:
@kayousef Can you help fill up these descriptions?

Reply:
@assafi done